test/test.py

```diff
@@ -763,6 +764,26 @@ def test_name_substitution(self):
             with_slash_after = set(f1 for f1 in files_after if '/' in f1)
             self.assertEqual(expected_slash_after, with_slash_after)
 
+    def test_project_accessions(self):
+        uuid = 'eca05046-3dad-4e45-b86c-8720f33a5dde'
+        version = '2019-03-17T220646.332108Z'
+        manifest, metadata_files = self._canned_bundle('staging', uuid, version)
+
+        # Append an additional value here to assert that multiple Accessions
+        # with the same type property are possible.
+        metadata_files['project_0.json']['insdc_project_accessions'].append('SRP000001')
+        expected_accessions = []
+        for k, v in metadata_files['project_0.json'].items():
+            if k.endswith('_accessions'):
+                t_ = k.strip('_accessions')[0]
+                for v_ in v:
+                    expected_accessions.append(Accession(t_, v_))
```

Comment:
There is a typo where `k.strip` is intended to be `k.split` however the test still passes due to the lack of member variables in the dataclass, which is causing the test below to compare `Accession()` equals `Accession()` (ignoring what is or isn't in each).

Instead of dynamically gathering the expected output I suggest simply hard coding the expected values in the assert.